utils/package-lock.json

```diff
@@ -0,0 +1,7263 @@
+{
```

Comment:
dont think we want to commit this, we have yarn.lock.

Reply:
I think the package-lock.json file got created when I ran either sanitize-dashboards or validate_quickstarts so I could better understand why the unit test was rejecting my dashboard JSON.  I removed that file from the repo.  

I don't have enough experience with Node.JS to know the impact of adding package-lock.json to the .gitignore, which would be  a more elegant and long-lasting fix.